src/domain/newsletter/SubscribeNewsletterPage.tsx

```diff
@@ -0,0 +1,97 @@
+import axios from 'axios';
+import { FormikHelpers } from 'formik';
+import { Notification } from 'hds-react';
+import * as React from 'react';
+import { useTranslation } from 'react-i18next';
+import { toast } from 'react-toastify';
+
+import { convertSubscribeFormData } from '../../clients/gruppo/lib/subscribers';
+import TextWithLineBreaks from '../../common/components/textWithLineBreaks/TextWithLineBreaks';
+import Container from '../app/layout/Container';
+import PageWrapper from '../app/layout/PageWrapper';
+import { ROUTES } from '../app/routes/constants';
+import { NewsletterSubscribeFormFields } from './newsletterSubscribeForm/constants';
+import NewsletterSubscribeForm, {
+  defaultInitialValues,
+} from './newsletterSubscribeForm/NewsletterSubscribeForm';
+import styles from './subscribeNewsletterPage.module.scss';
+
+const SubscribeNewsletterPage: React.FC = () => {
+  const { t } = useTranslation();
+  const [notification, setNotification] = React.useState<JSX.Element | null>(
+    null
+  );
+  const [notificationAutoCloseOpen, setNotificationAutoCloseOpen] =
+    React.useState(false);
```

Comment:
I think we shouldn't put react components to the state (it works but we could have simpler solution). I think we could have one state with `'success'` and `'error'` states, `null` would mean that notification is closed. Then render Notificarion based on the state